web-console/knikubevirt/NoStorageClassAvailable/NoStorageClassAvailable.md

```diff
@@ -0,0 +1,35 @@
+# Storage class isn’t defined
+
+https://github.com/openshift/openshift-origin-design/issues/249
+https://jira.coreos.com/browse/CNV-2842
+
+Currently in our designs we have a default storage class used when a disk is added in the Create VM flow, but we do not cover the absence of a default storage class.
+This PR should cover the state of no storage class selected and determine if/how the user should be notified.
+
+During the process of creating the VM, in the Storage step, the user needs to add a disk.
+
+![Add disk action](img/Add-disk-default.png)
+
+Clicking the ‘Add disk’ button will open a modal, where the user is asked to fill in the disk’s details.
+Since the Storage class isn’t available, and also no PVC that matches their requirements, the user will be notified with a red note below the Storage class dropdown that no storage class is available and they need to create one.
+
+We want to allow the user to set the Storage class within the wizard, with no need to be taken out of it.
+
+![Fill in disk details](img/Add-disk-modal-details-required.png)
+
+![Add disk modal with create storage class](img/Add-disk-modal-with-create-sc.png)
```

Comment:
For the storage class dropdown it shouldn't be disabled (grey) and the placeholder text should be "Select storage class" this is inline with the rest of the wizard.